vector/src/main/java/im/vector/app/features/roommemberprofile/devices/DeviceListBottomSheetViewModel.kt

```diff
@@ -61,23 +62,19 @@ class DeviceListBottomSheetViewModel @AssistedInject constructor(
 
     companion object : MavericksViewModelFactory<DeviceListBottomSheetViewModel, DeviceListViewState> by hiltMavericksViewModelFactory() {
 
-        override fun initialState(viewModelContext: ViewModelContext): DeviceListViewState? {
+        override fun initialState(viewModelContext: ViewModelContext): DeviceListViewState {
             val args = viewModelContext.args<DeviceListBottomSheet.Args>()
             val userId = args.userId
             val session = EntryPoints.get(viewModelContext.app(), SingletonEntryPoint::class.java).activeSessionHolder().getActiveSession()
-            return session.getUser(userId)?.toMatrixItem()?.let {
-                DeviceListViewState(
-                        userId = userId,
-                        allowDeviceAction = args.allowDeviceAction,
-                        userItem = it,
-                        isMine = userId == session.myUserId
-                )
-            } ?: return super.initialState(viewModelContext)
```

Comment:
Defaulting to super.initialState(viewModelContext) make the app crash because there is no default constructor for `DeviceListViewState`